webscrape.py

```diff
@@ -0,0 +1,42 @@
+from bs4 import BeautifulSoup
+import requests
+import smtplib
+
+
+
+def webscrape():
+    url = 'http://www.faa.gov/air_traffic/flight_info/aeronav/digital_products/vfr/?viewType=Print&viewClass=Print'
+    response = requests.get(url)
+    html = response.text
+    soup = BeautifulSoup(html, 'lxml')
+    table = soup.find(class_='striped')
+
+    try:
+        table.find_all('tr')
```

Comment:
Is this really the only line that could cause an exception? We're probably want the for loop below also wrapped in the try catch